### PR TITLE
fix multiple content linter warnings

### PR DESCRIPTION
**Pull request by nkaaf**
## What This PR Changes
- rename some Wi-Fi Boards to have consistent names
- fix LoRa® / LoRaWAN® trademarks
- fix spaces between value and unit
- fix unit names

## Contribution Guidelines
- [X] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/01.mkr/01.boards/mkr-nb-1500/tutorials/nb-scan-network/nb-scan-network.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ The goals of this project are:
 
 ## The uBlox SARA-R4 Module
 
-As every other MKR family board, the MKR NB 1500 board has a specific module for connectivity. It is called uBlox SARA-R4, and is designed to communicate over LTE Cat M1 or NB IoT networks with a speed of up to 375 kb/s.
+As every other MKR family board, the MKR NB 1500 board has a specific module for connectivity. It is called uBlox SARA-R4, and is designed to communicate over LTE Cat M1 or NB IoT networks with a speed of up to 375 KBps.
```
Comment (marqdevx) [inline content/hardware/01.mkr/01.boards/mkr-nb-1500/tutorials/nb-scan-network/nb-scan-network.md:46]: Should it be Bytes or Bits? 🤔

File: content/hardware/01.mkr/01.boards/mkr-wan-1310/tutorials/lora-message/lora-message.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ tags:
  - LoRa®
 author: 'Karl Söderby'
 libraries: 
-  - name: LoRa
+  - name: LoRa®
```
Comment (marqdevx) [inline content/hardware/01.mkr/01.boards/mkr-wan-1310/tutorials/lora-message/lora-message.md:11]: Thats the library name, so we should sync with the library's name
```suggestion
  - name: LoRa

```

File: content/hardware/01.mkr/01.boards/mkr-wan-1310/tutorials/lora-send-and-receive/lora-send-and-receive.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ tags:
  - LoRa®
 author: 'Karl Söderby'
 libraries: 
-  - name: LoRa
+  - name: LoRa®
```
Comment (marqdevx) [inline content/hardware/01.mkr/01.boards/mkr-wan-1310/tutorials/lora-send-and-receive/lora-send-and-receive.md:11]: Thats the library name, so we should sync with the library's name
```suggestion
  - name: LoRa
```

File: content/hardware/01.mkr/01.boards/mkr-wan-1310/tutorials/lora-send-and-receive/lora-send-and-receive.md [hunk 4]
```diff
@@ -59,7 +59,7 @@ ___
 -   2x Micro USB cable 
 -   Arduino IDE (offline and online versions available)
 -   Arduino SAMD core installed, [follow this link for instructions](https://www.arduino.cc/en/Guide/MKRWiFi1010#installing-drivers-for-the-mkr-wifi-1010)
--   LoRa library installed, see the [github repository](https://github.com/sandeepmistry/arduino-LoRa)
+-   LoRa® library installed, see the [github repository](https://github.com/sandeepmistry/arduino-LoRa)
```
Comment (marqdevx) [inline content/hardware/01.mkr/01.boards/mkr-wan-1310/tutorials/lora-send-and-receive/lora-send-and-receive.md:62]:
```suggestion
-   LoRa library installed, see the [github repository](https://github.com/sandeepmistry/arduino-LoRa)

```

File: content/hardware/01.mkr/01.boards/mkr-wan-1310/tutorials/lora-sensor-data/lora-sensor-data.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ tags:
   - Environmental data
 author: 'Karl Söderby'
 libraries: 
-  - name: LoRa
+  - name: LoRa®
```
Comment (marqdevx) [inline content/hardware/01.mkr/01.boards/mkr-wan-1310/tutorials/lora-sensor-data/lora-sensor-data.md:12]: Thats the library name, so we should sync with the library's name
```suggestion
  - name: LoRa
```

File: content/hardware/01.mkr/01.boards/mkr-wan-1300/tutorials/lora-button-press/lora-button-press.md [hunk 6]
```diff
@@ -9,7 +9,7 @@ tags:
   - LED
 author: 'Karl Söderby'
 libraries: 
-  - name: LoRa
+  - name: LoRa®
```
Comment (marqdevx) [inline content/hardware/01.mkr/01.boards/mkr-wan-1300/tutorials/lora-button-press/lora-button-press.md:12]: Thats the library name, so we should sync with the library's name
```suggestion
  - name: LoRa
```

**Review by marqdevx — CHANGES_REQUESTED** [review]

**Comment by marqdevx** [comment]
@nkaaf Please check out my suggestions 😄

**Comment by nkaaf** [comment]
Sorry for not replying to you. I was not at home last week, but I made the corrections locally. Now I have the time to push them. I thank you for your patience.

File: content/hardware/01.mkr/01.boards/mkr-wan-1310/tutorials/lora-message/lora-message.md (hunk 2 above)
Comment (nkaaf) [reply content/hardware/01.mkr/01.boards/mkr-wan-1310/tutorials/lora-message/lora-message.md]: changed

File: content/hardware/01.mkr/01.boards/mkr-wan-1310/tutorials/lora-send-and-receive/lora-send-and-receive.md (hunk 3 above)
Comment (nkaaf) [reply content/hardware/01.mkr/01.boards/mkr-wan-1310/tutorials/lora-send-and-receive/lora-send-and-receive.md]: changed

File: content/hardware/01.mkr/01.boards/mkr-wan-1310/tutorials/lora-send-and-receive/lora-send-and-receive.md (hunk 4 above)
Comment (nkaaf) [reply content/hardware/01.mkr/01.boards/mkr-wan-1310/tutorials/lora-send-and-receive/lora-send-and-receive.md]: changed

File: content/hardware/01.mkr/01.boards/mkr-wan-1310/tutorials/lora-sensor-data/lora-sensor-data.md (hunk 5 above)
Comment (nkaaf) [reply content/hardware/01.mkr/01.boards/mkr-wan-1310/tutorials/lora-sensor-data/lora-sensor-data.md]: changed

File: content/hardware/04.pro/boards/portenta-h7/tutorials/over-the-air-update/content.md [hunk 7]
```diff
@@ -151,7 +151,7 @@ We are using `OTA_Usage_Portenta.ino.PORTENTA_H7_M7` as a sketch name and for fa
 ### QSPI Storage Mode
 
 #### Setting Up
-To use internal **QSPI** storage for downloading the binary file via OTA (Over-The-Air), you will only need the Arduino Portenta H7 board connected to the computer with the [Arduino IDE](https://www.arduino.cc/en/software). With it, you will need to have selected the **Arduino Portenta H7 (M7 Core)** with the Flash split of **1MB M7 + 1MB M4** for the purpose of this tutorial and the corresponding port. 
+To use internal **QSPI** storage for downloading the binary file via OTA (Over-The-Air), you will only need the Arduino Portenta H7 board connected to the computer with the [Arduino IDE](https://www.arduino.cc/en/software). With it, you will need to have selected the **Arduino Portenta H7 (M7 Core)** with the Flash split of **1MB M7 + 1 MB M4** for the purpose of this tutorial and the corresponding port.
```
[the remark this answers is not in the document]
Comment (nkaaf) [reply content/hardware/04.pro/boards/portenta-h7/tutorials/over-the-air-update/content.md]: changed

File: content/hardware/04.pro/boards/portenta-h7/tutorials/over-the-air-update/content.md [hunk 8]
```diff
@@ -171,7 +171,7 @@ To use the **SD card** as the preferred OTA (Over-The-Air) storage device, we wi
 
 ![Arduino Portenta H7 with Vision Shield (Ethernet)](assets/portenta_h7_plus_vision_shield.svg)
 
-With this, you will need the Arduino Portenta H7 board connected to the computer with the Arduino IDE. You will need to have selected the **Arduino Portenta H7 (M7 Core)** with the Flash split of **1MB M7 + 1MB M4** for the purpose of this tutorial and the corresponding port.
+With this, you will need the Arduino Portenta H7 board connected to the computer with the Arduino IDE. You will need to have selected the **Arduino Portenta H7 (M7 Core)** with the Flash split of **1MB M7 + 1 MB M4** for the purpose of this tutorial and the corresponding port.
```
[the remark this answers is not in the document]
Comment (nkaaf) [reply content/hardware/04.pro/boards/portenta-h7/tutorials/over-the-air-update/content.md:174]: changed

File: content/hardware/01.mkr/01.boards/mkr-wan-1300/tutorials/lora-button-press/lora-button-press.md (hunk 6 above)
Comment (nkaaf) [reply content/hardware/01.mkr/01.boards/mkr-wan-1300/tutorials/lora-button-press/lora-button-press.md]: changed

File: content/hardware/01.mkr/01.boards/mkr-wan-1300/tutorials/lora-message/lora-message.md [hunk 9]
```diff
@@ -8,7 +8,7 @@ tags:
   - LoRa®
 author: 'Karl Söderby'
 libraries: 
-  - name: LoRa
+  - name: LoRa®
```
[the remark this answers is not in the document]
Comment (nkaaf) [reply content/hardware/01.mkr/01.boards/mkr-wan-1300/tutorials/lora-message/lora-message.md]: changed

File: content/hardware/01.mkr/01.boards/mkr-wan-1300/tutorials/lora-send-and-receive/lora-send-and-receive.md [hunk 10]
```diff
@@ -8,7 +8,7 @@ tags:
   - LoRa®
 author: 'Karl Söderby'
 libraries: 
-  - name: LoRa
+  - name: LoRa®
```
[the remark this answers is not in the document]
Comment (nkaaf) [reply content/hardware/01.mkr/01.boards/mkr-wan-1300/tutorials/lora-send-and-receive/lora-send-and-receive.md]: changed

File: content/hardware/01.mkr/01.boards/mkr-wan-1300/tutorials/lora-send-and-receive/lora-send-and-receive.md [hunk 11]
```diff
@@ -57,7 +57,7 @@ ___
 -   2x Micro USB cable .
 -   Arduino IDE (offline and online versions available).
 -   Arduino SAMD core installed ([follow this link for instructions](/content/software/ide-v1/installing-samd21-core)).
--   LoRa library installed (see the [github repository](https://github.com/sandeepmistry/arduino-LoRa)).
+-   LoRa® library installed (see the [github repository](https://github.com/sandeepmistry/arduino-LoRa)).
```
[the remark this answers is not in the document]
Comment (nkaaf) [reply content/hardware/01.mkr/01.boards/mkr-wan-1300/tutorials/lora-send-and-receive/lora-send-and-receive.md]: changed

File: content/hardware/01.mkr/01.boards/mkr-wan-1300/tutorials/lora-sensor-data/lora-sensor-data.md [hunk 12]
```diff
@@ -9,7 +9,7 @@ tags:
   - Environmental data
 author: 'Karl Söderby'
 libraries: 
-  - name: LoRa
+  - name: LoRa®
```
[the remark this answers is not in the document]
Comment (nkaaf) [reply content/hardware/01.mkr/01.boards/mkr-wan-1300/tutorials/lora-sensor-data/lora-sensor-data.md]: changed

File: content/hardware/01.mkr/01.boards/mkr-wan-1310/tutorials/lora-button-press/lora-button-press.md [hunk 13]
```diff
@@ -9,7 +9,7 @@ tags:
  - LED
 author: Karl Söderby
 libraries: 
-  - name: LoRa
+  - name: LoRa®
```
[the remark this answers is not in the document]
Comment (nkaaf) [reply content/hardware/01.mkr/01.boards/mkr-wan-1310/tutorials/lora-button-press/lora-button-press.md]: changed

File: content/hardware/01.mkr/01.boards/mkr-nb-1500/tutorials/nb-scan-network/nb-scan-network.md (hunk 1 above)
Comment (nkaaf) [reply content/hardware/01.mkr/01.boards/mkr-nb-1500/tutorials/nb-scan-network/nb-scan-network.md:46]: According to the Datasheet which is published on [content.arduino.cc](https://content.arduino.cc/assets/Arduino_SARA-R4_DataSheet_%28UBX-16024152%29.pdf#page=8) the data rate of "LTE category M1" is given in kb/s (kilobytes per second). This datasheet has the revision 18. On the internet I found a newer version (R-29) of the [datasheet](https://content.u-blox.com/sites/default/files/SARA-R4_DataSheet_UBX-16024152.pdf#page=12). There are some data rates which are given in kb/s and some are given in kbits/s (kilobits per second). But the interesting one (LTE category M1) is given in kb/s. So I think that bytes are correct.
If you have another opinion or another source please write a comment :)

Comment (marqdevx) [reply content/hardware/01.mkr/01.boards/mkr-nb-1500/tutorials/nb-scan-network/nb-scan-network.md:46]: In the datasheet that you've shared it says 375 kb/s (kilo bits) so I assume your change is not needed.

image

Also if the datasheet that we've shared is not up to date I will update that add it 👍

Comment (nkaaf) [reply content/hardware/01.mkr/01.boards/mkr-nb-1500/tutorials/nb-scan-network/nb-scan-network.md:46]: The data sheet (revision 18) mentions 375kb/s, the newer data sheet (revision 29) also 375kb/s. But the point is that in the column to the right of the 375kb/s (in the new datasheet) there is an indication in kbit/s. So why does the datasheet say kbit/s and kb/s when it should mean the same thing. But if kilobit is correct, then I need to change kb/s to kbps to satisfy the linter rule.
FYI: The image you have shared is from the old data sheet.

Comment (marqdevx) [reply content/hardware/01.mkr/01.boards/mkr-nb-1500/tutorials/nb-scan-network/nb-scan-network.md:46]: Screenshot from https://content.u-blox.com/sites/default/files/SARA-R4_DataSheet_UBX-16024152.pdf (R2)
Yes please change it to kbps and lets merge it! 🙌

Comment (marqdevx) [reply content/hardware/01.mkr/01.boards/mkr-nb-1500/tutorials/nb-scan-network/nb-scan-network.md:46]:
```suggestion
As every other MKR family board, the MKR NB 1500 board has a specific module for connectivity. It is called uBlox SARA-R4, and is designed to communicate over LTE Cat M1 or NB IoT networks with a speed of up to 375 kbps.
```

**Review by marqdevx — APPROVED** [review]

**Comment by marqdevx** [comment]
Let me fix the conflicts now 👁️ 
1 sec @nkaaf

**Comment by nkaaf** [comment]
Perfect, thank you! :star2:

**Comment by marqdevx** [comment]
Thank you so much for the help @nkaaf ❤️ 
Have a nice one!

**Pull request merged**